lib/modules/csv_exporter.rb

```diff
@@ -1,32 +1,180 @@
+        result.find_answer_by_key_value("short_name", "ecosystem_goal", default),
+        result.find_answer_by_key_value("short_name", "economic_arguments", default),
+        result.find_answer_by_key_value("short_name", "displace_motivations", default),
+        result.find_answer_by_key_value("short_name", "giving_voice_improves", default),
+        result.find_answer_by_key_value("short_name", "reform_global_trade", default),
+        result.find_answer_by_key_value("short_name", "Non_native_species", default),
+        result.find_answer_by_key_value("short_name", "impact_incomes", default),
+        result.find_answer_by_key_value("short_name", "support_corporations", default),
+        result.find_answer_by_key_value("short_name", "seek_no_harm", default),
+        result.find_answer_by_key_value("short_name", "giving_voice_ethical", default),
+        result.find_answer_by_key_value("short_name", "economic_growth", default),
+        result.find_answer_by_key_value("short_name", "communities_manage", default),
+        result.find_answer_by_key_value("short_name", "win_win", default),
+        result.find_answer_by_key_value("short_name", "rationales_weakens", default),
+        result.find_answer_by_key_value("short_name", "emphasise_value", default),
+        result.find_answer_by_key_value("short_name", "doom_gloom", default),
+        result.find_answer_by_key_value("short_name", "biological_diversity", default),
+        result.find_answer_by_key_value("short_name", "severe_perturbations", default),
+        result.find_demographic_answer_by_key("age", default),
+        result.find_demographic_answer_by_key("gender", default),
+        result.find_demographic_answer_by_key("education", default),
+        result.find_demographic_answer_by_key("educational_specialism", default),
+        result.find_demographic_answer_by_key("country", default),
+        self.format_multiple_answer(result.find_demographic_answer_by_key("location_of_work"), default),
+        self.format_multiple_answer(result.find_demographic_answer_by_key("conservation_sectors"), default),
+        result.find_demographic_answer_by_key("experience_in_other_fields", default),
+        self.format_multiple_answer(result.find_demographic_answer_by_key("non_conservation_sectors"), default),
+        result.find_demographic_answer_by_key("professional_engagement", default),
+        result.find_demographic_answer_by_key("seniority_of_current_role", default),
+        result.find_demographic_answer_by_key("category_of_most_professional_work", default),
+        result.find_demographic_answer_by_key("researcher_experience", default),
+        self.format_multiple_answer(result.find_demographic_answer_by_key("value_shaping_items"), default),
+        result.find_demographic_answer_by_key("shaping_values", default),
+        result.find_demographic_answer_by_key("email", default)
+      ].flatten
+
+      row.each do |answer|
```

Comment:
If I understand this snippet correctly, I think this should be a `map` instead of an `each`. `answer.gsub(",", "")` will be discarded, as the answers are not actually changing. 

Remember to cover both cases though, or the new array will have `nil` items where the `answer` is not a `String` 😁